lib/bootleg/mix_task.ex

```diff
@@ -9,14 +9,22 @@ defmodule Bootleg.MixTask do
       @spec run(OptionParser.argv) :: :ok
       if is_atom(unquote(task)) && unquote(task) do
         def run(args) do
-          Config.env(List.first(args) || :production)
```

Comment:
We already set a default `production` enviroment in `Config.env/1` so the setting of defaults here is unnecessary I think.

Reply:
yeah, agreed. in fact, when i was running into the "environment config not set" error last week in our test suite after paul's IO refactoring it actually took me a bit to find this default that was causing it because it's so buried.